shared/src/main/scala/mouse/boolean.scala

```diff
@@ -14,7 +14,9 @@ class ApplyIfPartiallyApplied[A](b: Boolean, a: A) {
 
 final class BooleanOps(private val b: Boolean) extends AnyVal {
 
-  @inline def option[A](a: => A): Option[A] = fold(Some(a), None)
+  @inline def option[A](a: => A): Option[A] = alternative[Option](a)
+
+  @inline def alternative[F[_]]: AlternativePartiallyApplied[F] = new AlternativePartiallyApplied[F](b)
```

Comment:
Though the `alternative` is a shorter variant, it's not so clear what it actually does. `option` method has the return type `Option[A]`. So, maybe `liftTo` is a better one?

Reply:
To be honest, the current `liftTo` method is custom in their behavior. I'd expected that method will produce the `F[A]` value, like in `cats` Option syntax.

Reply:
By and large, all ops on `Boolean` can be (and it does mostly) true- and false- based, like `valueOrZero` and `zeroOrValue`. I think that right approach in naming/providing functionality. It's more user-friendly than asking users to use the `unary_!`.

Reply:
There more methods that have a name based on the return type: `option`, `either`, `validatedNel`, `eitherNel`. When you generalize like `option` to arbitrary typeclass instances, the obvious way to do it is to name the method after the typeclass. @benhutchison seems to agree, and I'm honestly not willing to revert this change. The previous name is just too clunky.

Reply:
But method `alternative` is not returning the `Alternative` like those methods you mentioned (`option`, `either`, `validatedNel`, `eitherNel`). Because `Alternative` is not a data type (it's a type class), it seems a bit odd to me. 
But yeah, we haven't a unified convention about naming in the ops for now. Probably, it's a field that we should cover in the next major release (and provide automatic migrations via Steward).

Reply:
I think I've made my position clear: I'm not willing to revert this change. Take it or leave it.